sdk/keyvault/keyvault-secrets/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Fixed [bug 8378](https://github.com/Azure/azure-sdk-for-js/issues/8378), which caused the challenge based authentication to re-authenticate on every new request.
 - Fixed [bug 9005](https://github.com/Azure/azure-sdk-for-js/issues/9005), which caused parallel requests to throw if one of them needed to authenticate.
+- Fixed a bug in which the `keyId` was missing on the secret properties.
+- Removed the dependency of the TypeScript types for the `dom`.
```

Comment:
Before this specific change: https://github.com/Azure/azure-sdk-for-js/pull/9639/files#diff-f3c93f553a685134bd5ce75e2fbad25eR961 keyId wouldn't be populated, because none of the previous objects had this property.